generator/src/main/java/com/scottlogic/deg/generator/fieldspecs/FieldSpec.java

```diff
@@ -37,8 +41,20 @@ public static FieldSpec fromList(DistributedList<Object> whitelist) {
     public static FieldSpec fromRestriction(TypedRestrictions restrictions) {
         return new FieldSpec(null, restrictions, true, Collections.emptySet());
     }
-    public static FieldSpec empty() {
-        return new FieldSpec(null, null, true, Collections.emptySet());
+    public static FieldSpec fromType(Types type) {
+        TypedRestrictions defaultRestriction;
+        switch (type) {
+            case NUMERIC:
+                defaultRestriction = createDefaultNumericRestrictions(); break;
```

Comment:
would be nicer to just return on these lines

Reply:
Or extract out the `switch` to a new method, and return the default restriction.